Cargo.lock

```diff
@@ -1058,6 +1058,21 @@ dependencies = [
  "libloading 0.7.2",
 ]
 
+[[package]]
```

Comment:
Why do we pull in clap V2 and V3?

Reply:
That's a good question, I might have wrongly resolved a merge conflict. Will double check